README.md

```diff
@@ -32,6 +38,12 @@ docker run -it -v $srcDir:/src  <DOCKER_NAME>:<DOCKER_VERSION>
 We use the [codacy-plugins-test](https://github.com/codacy/codacy-plugins-test) to test our external tools integration.
 You can follow the instructions there to make sure your tool is working as expected.
 
+## Update tool version
+
+1. Open package.json file 
+2. Update *stylint* to the desired version
+3. Generate docs
```

Comment:
You can use `npm update` (please confirm) to update the versions.
You can even speak about dependabot here.

Reply:
Done ✅